src/ProductTour/Checkpoint.jsx

```diff
@@ -95,8 +96,13 @@ const Checkpoint = React.forwardRef(({
       role="dialog"
       style={{ visibility: checkpointVisible ? 'visible' : 'hidden', pointerEvents: checkpointVisible ? 'auto' : 'none' }}
     >
-      {/* This text is not translated due to Paragon's lack of i18n support */}
-      <span className="sr-only">Top of step {index + 1}</span>
+      <span className="sr-only">
+        <FormattedMessage
+          id="pgn.checkpoint.sr-only.message"
+          defaultMessage={`Top of step ${index + 1}`}
+          description="Screen-reader message to indicate the user's position in a sequence of checkpoints."
+        />
```

Comment:
This not the right way to translate messages with variables in it, the correct syntax is as follows (see [docs](https://formatjs.io/docs/react-intl/components#usage)):

```js
<FormattedMessage
  id="pgn.ProductTour.Checkpoint.position-text" // should change the id as well, we try to follow <component>.<subComponent> pattern in our ids
  defaultMessage="Top of step {step}"
  value={{ step: index + 1 }}
  description="Screen-reader message to indicate the user's position in a sequence of checkpoints."
/>
```